config/config.js

```diff
@@ -0,0 +1,10 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+module.exports = {
```

Comment:
I wonder about this vs. just letting people use `process.env` everywhere.  If we go with this approach, then every file has to `require` this file.  If we leave it all on `process.env.*` we don't.

I'd suggest we remove this and export nothing from `config.js`, and just make sure we `require('./config')` early in the code.

Reply:
ok, do you want me to add 
```
const dotenv = require('dotenv');
dotenv.config();
```
 at the very beginning of index.js for now ?